2.5/Dockerfile.rhel7

```diff
@@ -0,0 +1,63 @@
+FROM rhscl/s2i-base-rhel7:1
+
+# This image provides a Ruby environment you can use to run your Ruby
+# applications.
+
+EXPOSE 8080
+
+ENV RUBY_MAJOR_VERSION 2
+ENV RUBY_MINOR_VERSION 5
+ENV RUBY_VERSION "${RUBY_MAJOR_VERSION}.${RUBY_MINOR_VERSION}"
+ENV RUBY_SCL_NAME_VERSION "${RUBY_MAJOR_VERSION}${RUBY_MINOR_VERSION}"
+ENV RUBY_SCL "rh-ruby${RUBY_SCL_NAME_VERSION}"
+ENV RUBY_NODEJS_SCL "rh-nodejs6"
+ENV IMAGE_NAME "rhscl/ruby-${RUBY_SCL_NAME_VERSION}-rhel7"
+
+ENV SUMMARY="Platform for building and running Ruby $RUBY_VERSION applications" \
+    DESCRIPTION="Ruby $RUBY_VERSION available as docker container is a base platform for \
+building and running various Ruby $RUBY_VERSION applications and frameworks. \
+Ruby is the interpreted scripting language for quick and easy object-oriented programming. \
+It has many features to process text files and to do system management tasks (as in Perl). \
+It is simple, straight-forward, and extensible."
+
+LABEL summary="$SUMMARY" \
+      description="$DESCRIPTION" \
+      io.k8s.description="$DESCRIPTION" \
+      io.k8s.display-name="Ruby ${RUBY_VERSION}" \
+      io.openshift.expose-services="8080:http" \
+      io.openshift.tags="builder,ruby,ruby${RUBY_SCL_NAME_VERSION},${RUBY_SCL}" \
+      com.redhat.component="${RUBY_SCL}-docker" \
+      name="${IMAGE_NAME}" \
+      version="${RUBY_VERSION}" \
+      usage="s2i build https://github.com/sclorg/s2i-ruby-container.git \
+--context-dir=${RUBY_VERSION}/test/puma-test-app/ ${IMAGE_NAME} ruby-sample-app" \
+      maintainer="SoftwareCollections.org <sclorg@redhat.com>"
+
+RUN yum install -y yum-utils
+RUN prepare-yum-repositories rhel-server-rhscl-7-rpms
+ENV INSTALL_PKGS=" \
+${RUBY_SCL} \
+${RUBY_SCL}-ruby-devel \
+${RUBY_SCL}-rubygem-rake \
+${RUBY_SCL}-rubygem-bundler \
+${RUBY_NODEJS_SCL} \
+${RUBY_NODEJS_SCL}-npm \
+"
+RUN yum install -y --setopt=tsflags=nodocs ${INSTALL_PKGS} && \
+    yum clean all -y
+RUN rpm -V $INSTALL_PKGS
```

Comment:
Same thing for the RUN commands.

Reply:
Do you prefer below style to minimize the formula?

```
RUN yum install -y --setopt=tsflags=nodocs ${INSTALL_PKGS} && \
    yum clean all -y && \
    rpm -V $INSTALL_PKGS
```

The reason to split by 2 `RUN`s is that it is good to check the command step by step when running `docker build`. Like this.

```
$ docker build --rm -f Dockerfile.rhel7 .
...
Step 17/23 : RUN yum install -y --setopt=tsflags=nodocs ${INSTALL_PKGS} &&     yum clean all -y
 ---> Using cache
 ---> b6012624881f
Step 18/23 : RUN rpm -V $INSTALL_PKGS
 ---> Using cache
 ---> ad585c5d93b7
...
```

Maybe I have same thought with RPM's `Dockferfile`.
https://github.com/rpm-software-management/rpm/blob/master/ci/Dockerfile
But you prefer to minimize it. I will change it.

It might be good to keep a consistent style with other `s2i-foo-container`.

Reply:
Other sclorg containers are using the style I recommended so it would be better if ruby used it as well.

Reply:
ok I will modify it.